app/config/locale/templates/email-base-cloud.tpl

```diff
@@ -0,0 +1,234 @@
+            <table>
+                <tr>
+                    <td>
+                        <img
+                            height="32px"
+                            src="https://appwrite.io/assets/logotype/white.png"
+                        />
+                    </td>
+                </tr>
+            </table>
+
+            <table style="margin-top: 32px">
+                <tr>
+                    <td>
+                        <h1>{{subject}}</h1>
+                    </td>
+                </tr>
+            </table>
+
+            <table style="margin-top: 40px">
+                <tr>
+                    <td>{{message}}</td>
+                </tr>
+            </table>
+
+            <table
+                style="
+                    padding-top: 32px;
+                    margin-top: 32px;
+                    border-top: solid 1px #e8e9f0;
+                "
+            >
+                <tr>
+                    <td></td>
+                </tr>
+            </table>
+
+            <table style="width: auto; margin: 0 auto">
+                <tr>
+                    <td style="padding-left: 4px; padding-right: 4px">
+                        <a
+                            href="https://twitter.com/appwrite"
+                            class="social-icon"
+                            title="Twitter"
+                        >
+                            <img src="https://appwrite.io/email/x.png" height="24" width="24" />
+                        </a>
+                    </td>
+                    <td style="padding-left: 4px; padding-right: 4px">
+                        <a
+                            href="https://appwrite.io/discord"
+                            class="social-icon"
+                        >
+                            <img src="https://appwrite.io/email/discord.png" height="24" width="24" />
+                        </a>
+                    </td>
+                    <td style="padding-left: 4px; padding-right: 4px">
+                        <a
+                            href="https://github.com/appwrite/appwrite"
+                            class="social-icon"
+                        >
+                            <img src="https://appwrite.io/email/github.png" height="24" width="24" />
+                        </a>
+                    </td>
+                </tr>
+            </table>
+            <table style="width: auto; margin: 0 auto; margin-top: 60px">
+                <tr>
+                    <td><a href="https://appwrite.io/terms">Terms</a></td>
+                    <td style="color: #e8e9f0">
+                        <div style="margin: 0 8px">|</div>
+                    </td>
+                    <td><a href="https://appwrite.io/privacy">Privacy</a></td>
+                </tr>
+            </table>
+            <p style="text-align: center" align="center">
+                &copy; 2023 Appwrite | 251 Little Falls Drive, Wilmington 19808,
```

Comment:
2024

Reply:
We can use a function here instead of manually updating.

Reply:
Agree, {{year}} placeholder sounds like good idea (with setParam)